Cannot initialize model with low cpu memory usage because `accelerate` was not found in the environment. Defaulting to `low_cpu_mem_usage=False`. It is strongly recommended to install `accelerate` for faster and less memory-intense model loading. You can do so with:
```
pip install accelerate
```
.
/Library/Frameworks/Python.framework/Versions/3.10/lib/python3.10/site-packages/diffusers/pipelines/pipeline_loading_utils.py:221: FutureWarning: You are loading the variant fp16 from stabilityai/stable-diffusion-2-1-base via `revision='fp16'` even though you can load it via `variant=`fp16`. Loading model variants via `revision='fp16'` is deprecated and will be removed in diffusers v1. Please use `variant='fp16'` instead.
  warnings.warn(
Loading pipeline components...:   0%|                                                                                  | 0/5 [00:00<?, ?it/s]An error occurred while trying to fetch /Users/navtegh/.cache/huggingface/hub/models--stabilityai--stable-diffusion-2-1-base/snapshots/1f758383196d38df1dfe523ddb1030f2bfab7741/unet: Error no file named diffusion_pytorch_model.safetensors found in directory /Users/navtegh/.cache/huggingface/hub/models--stabilityai--stable-diffusion-2-1-base/snapshots/1f758383196d38df1dfe523ddb1030f2bfab7741/unet.
Defaulting to unsafe serialization. Pass `allow_pickle=False` to raise an error instead.
Loading pipeline components...:  40%|█████████████████████████████▌                                            | 2/5 [00:13<00:16,  5.51s/it]An error occurred while trying to fetch /Users/navtegh/.cache/huggingface/hub/models--stabilityai--stable-diffusion-2-1-base/snapshots/1f758383196d38df1dfe523ddb1030f2bfab7741/vae: Error no file named diffusion_pytorch_model.safetensors found in directory /Users/navtegh/.cache/huggingface/hub/models--stabilityai--stable-diffusion-2-1-base/snapshots/1f758383196d38df1dfe523ddb1030f2bfab7741/vae.
Defaulting to unsafe serialization. Pass `allow_pickle=False` to raise an error instead.
Loading pipeline components...: 100%|██████████████████████████████████████████████████████████████████████████| 5/5 [00:14<00:00,  2.86s/it]
You have disabled the safety checker for <class 'inverse_stable_diffusion.InversableStableDiffusionPipeline'> by passing `safety_checker=None`. Ensure that you abide to the conditions of the Stable Diffusion license and do not expose unfiltered results in services or applications open to the public. Both the diffusers team and Hugging Face strongly recommend to keep the safety filter enabled in all public facing circumstances, disabling it only for use-cases that involve analyzing network behavior or auditing its results. For more information, please have a look at https://github.com/huggingface/diffusers/pull/254 .
Traceback (most recent call last):
  File "/Users/navtegh/Documents/Watermarking-diffusion-models/run_tree_ring_watermark.py", line 230, in <module>
    main(args)
  File "/Users/navtegh/Documents/Watermarking-diffusion-models/run_tree_ring_watermark.py", line 30, in main
    pipe = InversableStableDiffusionPipeline.from_pretrained(
  File "/Library/Frameworks/Python.framework/Versions/3.10/lib/python3.10/site-packages/huggingface_hub/utils/_validators.py", line 114, in _inner_fn
    return fn(*args, **kwargs)
  File "/Library/Frameworks/Python.framework/Versions/3.10/lib/python3.10/site-packages/diffusers/pipelines/pipeline_utils.py", line 948, in from_pretrained
    model = pipeline_class(**init_kwargs)
  File "/Users/navtegh/Documents/Watermarking-diffusion-models/inverse_stable_diffusion.py", line 45, in __init__
    super(InversableStableDiffusionPipeline, self).__init__(vae,
  File "/Users/navtegh/Documents/Watermarking-diffusion-models/modified_stable_diffusion.py", line 31, in __init__
    super(ModifiedStableDiffusionPipeline, self).__init__(vae,
  File "/Library/Frameworks/Python.framework/Versions/3.10/lib/python3.10/site-packages/diffusers/pipelines/stable_diffusion/pipeline_stable_diffusion.py", line 276, in __init__
    self.register_modules(
  File "/Library/Frameworks/Python.framework/Versions/3.10/lib/python3.10/site-packages/diffusers/pipelines/pipeline_utils.py", line 165, in register_modules
    library, class_name = _fetch_class_library_tuple(module)
  File "/Library/Frameworks/Python.framework/Versions/3.10/lib/python3.10/site-packages/diffusers/pipelines/pipeline_loading_utils.py", line 735, in _fetch_class_library_tuple
    library = not_compiled_module.__module__.split(".")[0]
AttributeError: 'bool' object has no attribute '__module__'
